archlinuxcn/llvm-git/lilac.yaml

```diff
@@ -0,0 +1,10 @@
+maintainers:
+  - github: solaraquarion
+
+build_prefix: extra-x86_64
+
+pre_build: vcs_update
+
+update_on:
```

Comment:
Should update on ocaml and python as well.

See `package_llvm-ocaml-julia` from `llvm-julia` for how you can add ocaml version dependency automatically.

Reply:
i did a bunch of change. hopefully it's better